vformer/encoder/cvt.py

```diff
@@ -0,0 +1,76 @@
+import torch
+import torch.functional as F
+import torch.nn as nn
+from timm.models.layers import DropPath
+
+from ..attention import VanillaSelfAttention
+from ..decoder import MLPDecoder
+from ..functional.norm import PreNorm
+from .nn import FeedForward
+
+
+class CVTEncoderBlock(nn.Module):
+    """
```

Comment:
Could you please a one-line description about the module here before listing the parameters?

Reply:
Could you please look and give feedback on the current description?
Thanks

Reply:
Yup, it looks good.

Reply:
I think you can remove the first sentence since that is common to all encoders. 
You can mention the main/salient feature of the class (if any). For eg, for the cross-attention encoder, it could be something like "Processes inputs at multiple scales and performs cross attention between them".